types/imagekit/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": "dtslint/dt.json"
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-redundant-jsdoc-2": false
```

Comment:
This rule crashes on `@see` right now, but shouldn't fail CI. Can you revert this for now?

(I merged a fix this morning but haven't shipped it yet.)